app/javascript/components/zone-form/zone-form.schema.js

```diff
@@ -0,0 +1,113 @@
+import { componentTypes } from '@@ddf';
+
+const createSchema = (props) => ({
+    fields: [
+        {
+            component: componentTypes.SUB_FORM,
+            name: 'zone-information-subform',
+            title: __('Zone Information'),
+            fields: [
+                {
+                    component: componentTypes.TEXT_FIELD,
+                    id: 'name',
+                    name: 'name',
+                    label: __('Name'),
+                    maxLength: 50,
+                },
+                {
+                    component: componentTypes.TEXT_FIELD,
+                    id: 'description',
+                    name: 'description',
+                    label: __('Description'),
+                    maxLength: 50,
+                },
+                {
+                    component: componentTypes.TEXT_FIELD,
+                    id: 'proxy_server_ip',
+                    name: 'settings.proxy_server_ip',
+                    label: __('SmartProxy Server IP'),
+                    maxLength: 50,
+                }         
+            ]
+        },
+        {
+            component: componentTypes.SUB_FORM,
+            name: 'ntp-servers-subform',
+            title: __('NTP Servers'),
+            description: __("* Specified NTP settings apply to this zone only and are not global."),    
+            fields: [
+                { 
+                    component: componentTypes.TEXT_FIELD,
+                    id: 'ntp_server_1',
+                    name: 'ntp_server_1',
+                    initialValue: "0.pool.ntp.org",
+                    maxLength: 50,  
```

Comment:
The `ViewHelper::MAX_NAME_LEN` constant in the original form is set to 255, so let's make this 255 too, on the fields below as well.